feature branch

rashidmvk opened the pull request:
PROJECT: APPLE PROJECT : 

-Added background image
-Added Navbar
-Fixed all the changes recommended by TSE.

krys2fa commented on README.md:
```diff
@@ -1,2 +1,40 @@
+- html & css,
+- Visual Studio Code, Git and Github.
+
+## Live Demo
+[Live Demo Link](https://rashidmvk.github.io/)
```
- [ ] [MANDATORY] The link to your live version of your project does not display the project.
![Screen Shot 2020-07-28 at 5 28 46 PM](https://user-images.githubusercontent.com/18187039/88700255-f094bb00-d0f7-11ea-999a-d4a086d631f1.png)

krys2fa commented:
### Status: Changes Required ♻️ 

Hello, @rashidmvk 

I am re-stating the changes requested earlier because it seems you missed some of them. They are pointed out below.

- [ ] [MANDATORY] You have linter errors in your project. Kindly fix this.
<b>Hint:</b> You can run this command `npx stylelint "**/*.{css,scss}" --fix` locally to fix a few of them. 

- [ ] [MANDATORY] HTML elements in your project should be aligned with float and  flex or grid ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/)) 
<b>Hint:</b> Float was not used in styling (some) elements.

- [ ] [MANDATORY] Your Pull Request should have a meaningful title and a brief description. Take a step back and look at the [3 basic rules to follow in Microverse projects.](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)

- [ ] [MANDATORY] The background image does not show in your page. Please fix this.

- [ ] [MANDATORY] The whole page does not fit in particular screen size, hence you need to scroll horizontally to see the whole page. Please fix this. 

- [ ] [OPTIONAL] Ensure that your README file looks professional by checking that image and web links are working fine.

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

rashidmvk commented:
Dear TSE,
I have completely restructured the html layout and css  stylings completely to fix all your concerns and thats it took sometime . Also I am attaching the screenshot of the page here to show how its is displaying  on my browser. Since this projcet don't have to be reesponsive, I didn't take care of that one. I hope this time my project got approved.

Following are the feature related to new code review request : 

PROJECT: APPLE PROJECT :

-Restructured the HTML layout
-Added background image
-Added Navbar
-Fixed all the changes recommended by TSE.

kitpao commented on css/styles.css:
```diff
@@ -0,0 +1,145 @@
+}
+
+.bg-image-area {
+  background-image: url(/your_verse_hero.png);
```
- [ ] Your image is not displaying, not in Firefox, not in Chrome, not in Edge. I guess the problem is that you are missing a dot (.) and the images folder in your path. Please add it as you did with the other images

kitpao commented on css/styles.css:
```diff
@@ -0,0 +1,145 @@
+.bg-image-area {
+  background-image: url(/your_verse_hero.png);
+  background-repeat: no-repeat;
+  background-size: contain;
```
- [ ] For your image to cover the screen, you should use "cover" instead of "contain"

kitpao commented on css/styles.css:
```diff
@@ -0,0 +1,145 @@
+  background-image: linear-gradient(to bottom, #626262, black);
+}
+
+.bg-image-area {
+  background-image: url(/your_verse_hero.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
```
- [ ] You will also need to use `background-position` attribute to put the image in the correct place

kitpao commented on css/styles.css:
```diff
@@ -0,0 +1,145 @@
+  display: flex;
+}
+
+.flex-container img {
+  width: 324px;
+}
```
- [ ] The problem with your images not taking all the width, is here. I would suggest you to remove this block of code, and instead, go to your nameless divs containing the `<a>'s` and add this rule `width: 25%;`

rashidmvk commented:
Following are the feature related to new code review request :

PROJECT: APPLE PROJECT :

-Fixed background image issue.
-Fixed flex container fit issue.
-Fixed all the changes recommended by TSE.

kubilaycaglayan commented on css/styles.css:
```diff
@@ -0,0 +1,152 @@
+  top: 150px;
+  width: 67%;
+  margin-bottom: 20px;
+  left: 410px;
```
This is where you are setting the horizontal position of the nav-bar. Instead of **left: 410px** property, try to implement a different approach to fix it into the center.

kitpao commented on css/styles.css:
```diff
@@ -0,0 +1,174 @@
+  top: 130px;
+  width: 70%;
+  margin-bottom: 20px;
+  left: 200px;
```
- [ ] **OPTIONAL** Another way to center it is doing the math, try to use % instead of px for positioning to avoid headaches
(screen_width -  nav_width)/2
(100% - 70%)/2 = 15%
=> `left: 15%;`
